VaticanBAV/et/BAVet190.xml

```diff
@@ -27,6 +27,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <collection>Aethiopici</collection>
                         <idno>Aeth. 190</idno>
                     </msIdentifier>
+
+                    <msContents>
+
+                      <msItem xml:id="ms_i1">
+                        <locus from="2r" to="7v" />
+                        <title type="incomplete" ref="LIT2915RepCh196" />
+                        <textLang mainLang="gez"/>
+                        <explicit xml:lang="gez">ወሰላም፡ ለመቃብሩ፨ ገብረ፡ መንፈስ፡ ቅዱስ፡ አቡየ፡ ለእግዚአብሔር፡ ፍቁሩ፨ ደመናት፡ አንኰርኰሩ፨ አዕዋፈ፡ ሰማይ፡ ዘመሩ፨ በጊዜ፡ ሞትከ፡ መላእክት፡ አንከሩ፨ ኦአምላከ፡ ገብረ፡ መንፈስ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/></explicit>
+                        <note>Incomplete at the end.</note>
```

Comment:
is it incomplete because it was copied thus, or because of material loss? we never finished the discussion https://github.com/BetaMasaheft/Documentation/issues/1161 as to whether the two types of incompleteness should be kept apart in the encoding, for the moment we do not, but it might be helpful if the information is at least provided as free text/note when possible

Reply:
The catalogue is quite lacking in this regard. It doesn't say anything explicit regarding lost text or give any information that would clarify it, such as differences in hands. That the quires and units don't correspond complicates the matter, unless the quires have folios stitched in, but the catalogue never includes such information.